extras/data/EMC/metadata.yaml

```
@@ -0,0 +1,33 @@
+
+arxiv: null
+hepdata: null
```

```suggestion
arxiv: null
hepdata: null
```
are there no references?

---

(similar for others)